src/main/java/reposense/model/CliArguments.java

```diff
@@ -9,11 +9,14 @@
  * Represents command line arguments user supplied when running the program.
  */
 public abstract class CliArguments {
+    protected Path configFolderPath;
     protected Path outputFilePath;
     protected Optional<Date> sinceDate;
     protected Optional<Date> untilDate;
     protected List<String> formats;
 
+    public Path getConfigFolderPath() { return configFolderPath; }
```

Comment:
```
public Path getConfigFolderPath() { return configFolderPath; }
```
->

```
public Path getConfigFolderPath() { 
    return configFolderPath;
}
```
Also, move this to `ConfigCliArguments`.

Reply:
Requested change has been made